paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/EventReporter.kt

```diff
@@ -169,6 +169,12 @@ internal interface EventReporter {
         error: Throwable,
     )
 
+    /**
+     * The customer cannot properly return from Link payments or other LPM payments using
+     * browser intents.
+     */
+    fun onCannotProperlyReturnFromLinkAndOtherLPMs()
```

Comment:
This seems like a good place to link the doc you wrote on what's going on. Will be nice for future readers to understand what's actually going on.